src/fsharp/symbols/Symbols.fs

```diff
@@ -75,12 +75,18 @@ module Impl =
          (fun (asmName, path) -> invalidOp (sprintf "The entity or value '%s' does not exist or is in an unresolved assembly. You may need to add a reference to assembly '%s'" path asmName))
          f
 
-    let makeReadOnlyCollection (arr: seq<'T>) = 
-        System.Collections.ObjectModel.ReadOnlyCollection<_>(Seq.toArray arr) :> IList<_>
-        
-    let makeXmlDoc (XmlDoc x) = makeReadOnlyCollection x
+    let makeReadOnlyCollection<'T> (items : 'T []) = 
+        System.Collections.ObjectModel.ReadOnlyCollection<'T>(items) :> IList<'T>
```

Comment:
Let's put an `open` at the top of the file for these (even though this is just augmenting existing code)